src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,84 @@
+package duke;
+
+import java.util.Scanner;
+
+import duke.components.FileStorage;
+import duke.components.Parser;
+import duke.components.TaskList;
+import duke.components.Ui;
+import duke.exception.FileCorruptedException;
+import duke.exception.FileLoadException;
+import duke.exception.FileNoExistingTasksException;
+
+/**
+ * Represents the main class where the chatbot will run.
+ */
+public class Duke {
+    private static String filePath = "./data/duke.txt";
+    private Scanner scanner = new Scanner(System.in);
+    private TaskList fullList;
+    private final FileStorage fileStorage;
+    private Parser parser;
+    private Ui ui;
+
+    /**
+     * Constructs a chatBot which will load any existing tasks stored in the filePath
+     * specified.
+     *
+     */
+    public Duke() {
+        this.fileStorage = new FileStorage(filePath);
+        try {
+            this.fullList = new TaskList(fileStorage.loadFiles());
+            this.ui = new Ui();
+            this.parser = new Parser(fileStorage, fullList, ui);
+        } catch (FileCorruptedException e) {
+            System.out.println("Saved tasks is corrupted. Please start adding new tasks");
+            this.fullList = new TaskList();
+            this.ui = new Ui();
+            this.parser = new Parser(filePath);
+        } catch (FileNoExistingTasksException e) {
+            System.out.println("No saved tasks! Please start adding new tasks");
+            this.fullList = new TaskList();
+            this.ui = new Ui();
+            this.parser = new Parser(filePath);
+        } catch (FileLoadException e) {
+            System.out.println("Error when reading saved tasks. Please start adding new tasks");
+            this.fullList = new TaskList();
+            this.ui = new Ui();
+            this.parser = new Parser(filePath);
+        }
+    }
+
+    public String getResponse(String input) {
+        return parser.createTaskAction(input);
+    }
+
+    public String getWelcome() {
+        return ui.showWelcome();
+    }
+
+    //    /**
+    //     * Initialises the application, displays a welcome message and
+    //     * enters a loop to process user inputs until the use enters "bye".
+    //     */
+    //    public void run() {
+    //        ui.showWelcome();
+    //        boolean isBye = false;
+    //        while (!isBye) {
+    //            String input = scanner.nextLine();
+    //            isBye = parser.isBye(input);
+    //            parser.createTaskAction(input);
+    //        }
+    //        scanner.close();
+    //    }
+
+    //    /**
+    //     * Entry point of the Duke application.
+    //     *
+    //     * @param args Command-line arguments. Not used in this context.
+    //     */
+    //    public static void main(String[] args) {
+    //        new Duke().run();
+    //    }
```

Comment:
Should this unused code be deleted instead of commented out to keep the code cleaner?